### PR TITLE
Dropdown values can be deselected.

### LINKED ISSUE #39: Question: Clear or set paper-dropdown-menu value
Hi,

I can't find a way to clear a paper-dropdown-menu. I would like to reset it to its initial state when my form is sent (ajax). I can't find anything in the demos or doc.

Also, is it possible the set the value. Exemple: pre-selected value.

Thanks a lot

### PR DESCRIPTION
**Pull request by cdata**
This is enabled by listening for a bubbling `iron-deselect` event and
setting the current selection to `null` when it is received.

Fixes #39

### CODE REVIEW

File: paper-dropdown-menu.html [hunk 1]
```diff
@@ -74,6 +77,12 @@
       text-align: left;
       cursor: pointer;
 
+      /* NOTE(cdata): Both values are needed, since some phones require the
+       * value to be `transparent`.
+       */
+      -webkit-tap-highlight-color: rgba(0,0,0,0);
+      -webkit-tap-highlight-color: transparent;
+
```
Comment (cdata) [inline paper-dropdown-menu.html]: This patch is orthogonal to the rest of this PR, but I figured I would sneak it in.

**Comment by notwaldorf** [comment]
lgtm 👍

**Comment by frankiefu** [comment]
LGTM :+1:

**Comment by cdata** [comment]
Thanks guys!
